### PR TITLE
clarify PR template text

### PR DESCRIPTION
**Pull request by kuhnchris**
...as per discussion with @thelostone-mc and @PixelantDesign

##### Description

Clarifies the PR text change to fix some wording/misunderstandings

##### Checklist

- [N/A] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [N/A] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [N/A] Linter status: 100% pass
- [N/A] Added relevant tests 
- [N/A] Tests pass and code coverage has not decreased
- [N/A] Changes don't break existing behavior
- [N/A] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [N/A] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [N/A] Completed: Code review by core team and any requested changes

##### Affected core subsystem(s)

doc

##### Refers/Fixes

N/A, Slack discussion

##### Testing and Sign-off

N/A

###### Contributor

- [N/A] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [N/A] Tested all changes **locally**
- [N/A] Verified existing functionality
- [N/A] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [N/A] CI tests and linting pass
- [N/A] No conflicts (migrations, files, etc)
- [N/A] Regression tested against staging

###### Funder

- [N/A] Validated requested changes were made to specification
- [N/A] Bounty payout released to the contributor

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -39,7 +41,7 @@
 
 <!-- Why should the PR reviewer trust that this change doesn't break anything? How have you tested this change? -->
 
-###### Contributor
+###### Contributor (PR submitter & devs working on this)
```
Comment (mbeacom) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do we need to provide clarity in the header of this section?

Comment (kuhnchris) [reply .github/PULL_REQUEST_TEMPLATE.md]: Kinda, I was confused at first, but we can drop it if you want

Comment (thelostone-mc) [reply .github/PULL_REQUEST_TEMPLATE.md]: Let's drop it ! ^_^

**Comment by kuhnchris** [comment]
@thelostone-mc done.

**Comment by thelostone-mc** [comment]
Thanks @kuhnchris  ^_^

**Comment by kuhnchris** [comment]
you're welcome. :-)

**Pull request merged**